apps/web/src/features/application/applicationSlice.ts

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit'
+import type { PayloadAction } from '@reduxjs/toolkit'
+import type { RootState } from '../../app/store'
+
+interface ApplicationState {
+  isTrue: boolean
+}
+
+const initialState: ApplicationState = {
+  isTrue: true,
+}
+
+export const applicationSlice = createSlice({
+  name: 'application',
+  initialState,
+  reducers: {
+    updateIsTrue: (state, action: PayloadAction<boolean>) => {
+      state.isTrue = action.payload
```

Comment:
Very nice. I appreciate that you were willing to refactor this twice. In the end I think you've produced the best solution. Take a picture of this code and put it on your refrigerator because this is elegant. 

Here's why this code is good:
- It's readable / semantic - I can look at this and immediately understand what you are doing
- It's terse - No extra characters or lines of code without sacrificing readability